client/reader/site-subscriptions-manager/comment-subscriptions-manager/comment-subscriptions-manager.tsx

```diff
@@ -1,7 +1,6 @@
 import { useTranslate } from 'i18n-calypso';
 import { Comments } from 'calypso/landing/subscriptions/components/tab-views';
 import SubscriptionsManagerWrapper from '../subscriptions-manager-wrapper';
-import '../style.scss';
```

Comment:
No need to import styles, it is being imported with the `SubscriptionManagerWrapper`.